db/seeds.rb

```diff
@@ -5,3 +5,28 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+User.create!(username:  "Iamtheadmin",
+             email: "zhwtf@gmail.com",
+             password:              "Zhwtf2567369!",
+             password_confirmation: "Zhwtf2567369!",
+             is_admin: true,
+             firstname: "hao",
+             lastname: "zheng")
+
+30.times do |n|
+  name  = Faker::Name.first_name
+  email = "example-#{n+1}@gmail.com"
```

Comment:
this is so not ok! you can't use gmail!!! if you want to add any example email, it needs to be @example.com